### ISSUE TITLE
Saving and restoring prototype seems to only save one dimension, and then causes an error placing new tokens

### ISSUE THREAD

**Issue by benbarbour** [unsoluble/quickscale#5]
**Foundry:** v10.286
**DnD5e:** v2.0.3

**Steps to Reproduce:**
1. Drop a token
2. Scale it with quick-scale
3. Save the prototype with quick-scale (default `SHIFT-\`)
4. Revert to prototype with quick-scale (default `\`)

**Expected Result:** The scale is restored
**Actual Result:** See screenshot; scale is restored in one dimension only.

![image](https://user-images.githubusercontent.com/487715/191860842-28948dc2-602e-45f0-9190-e493ca07dcfd.png)

After the error occurs (and my reproduction steps don't seem to get it 100% of the time :/), attempting to drop any other token on the scene results in this error, until Foundry is refreshed.

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clear')
[No packages detected]
    at Token5e._refreshTarget (foundry.js:45708:17)
    at Token5e._refresh (foundry.js:45601:10)
    at Token5e.refresh (foundry.js:25015:10)
    at Canvas.highlightObjects (foundry.js:23931:11)
    at DrawingsLayer.deactivate (foundry.js:30215:12)
    at TokenLayer.activate (foundry.js:30184:76)
    at TokenLayer._onDropActorData (foundry.js:36910:10)
```

After the refresh the token is still squashed, and further copies of the actor dropped on the scene are also squashed. Manually updating the token (or prototype token's) scale by dragging the bar in the settings window seems to resolve the issue, until it is triggered with quick-scale again.

**Comment by unsoluble** [comment]
Hmm! Weird. Will take a look tonight, thanks for the ticket.

**Comment by unsoluble** [comment]
Fixed in 1.5.3
